lessons/de/chapter_5.yaml

```
@@ -0,0 +1,224 @@
+
+
+    Wenn statische Lebenszeitressourcen Referenzen enthalten, müssen sie alle `'statisch` sein
+    (Alles andere würde nicht lange genug leben).
+
+
+    Speicherdetail:
+
+
+    * Das Ändern statischer Variablen ist von Natur aus gefährlich, weil sie es sind
+    weltweit zugänglich und für jedermann lesbar
+      Einführung der Möglichkeit eines Datenwettlaufs. Wir werden später über die Herausforderungen globaler Daten sprechen.
+    * Rust erlaubt die Verwendung von „unsafe { ... }“-Blöcken, um einige Operationen auszuführen
+    dass der Compiler keine Speichergarantien geben kann. Die [<span
+    style="color:red; Schriftstärke:
+    fett;">R̸͉̟͈͔̄͛̾̇͜U̶͓͖͋̅Ṡ̴͉͇̃̉̀T̵̻̻͔̟͉́͆Ơ̷̥̟̳̓͝N̶̨̼̹̲͛Ö̵̝͉̖̏̾̔M̶̡̠̺̠̐ ͜Î̷̛͓̣̃̐̏C̸̥̤̭̏͛̎͜O̶̧͚͖͔̊͗̇͠N̸͇̰̏̏̽̃</span>](https://doc.rust-lang.org/nomicon/)
+    darüber sollte nicht beiläufig gesprochen werden.
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=static%20PI%3A%20f64%20%3D%203.1415%3B%0A%0Afn%20main()%20%7B%0A%20%20%20%20%2F%2F%20variabilele%20statice%20pot%20fi%20%C3%AEncadrate%20%C3%AEntr-o%20func%C8%9Bie%0A%20%20%20%20static%20mut%20SECRET%3A%20%26%27static%20str%20%3D%20%22swordfish%22%3B%0A%0A%20%20%20%20%2F%2F%20string-urile%20%C3%AEntre%20ghilimele%20au%20durat%C4%83%20de%20via%C8%9B%C4%83%20static%C4%83%0A%20%20%20%20let%20msg%3A%20%26%27static%20str%20%3D%20%22Hello%20World!%22%3B%0A%20%20%20%20let%20p%3A%20%26%27static%20f64%20%3D%20%26PI%3B%0A%20%20%20%20println!(%22%7B%7D%20%7B%7D%22%2C%20msg%2C%20p)%3B%0A%0A%20%20%20%20%2F%2F%20Pute%C8%9Bi%20%C3%AEnc%C4%83lca%20unele%20reguli%2C%20dar%20trebuie%20s%C4%83%20fi%C8%9Bi%20explici%C8%9Bi%0A%20%20%20%20unsafe%20%7B%0A%20%20%20%20%20%20%20%20%2F%2F%20putem%20seta%20SECRET%20cu%20un%20string%20%C3%AEntre%20ghilimele%0A%20%20%20%20%20%20%20%20%2F%2F%20pentru%20c%C4%83%20are%20de%20asemenea%20durat%C4%83%20de%20via%C8%9B%C4%83%20static%C4%83%20(%60static)%0A%20%20%20%20%20%20%20%20SECRET%20%3D%20%22abracadabra%22%3B%0A%20%20%20%20%20%20%20%20println!(%22%7B%7D%22%2C%20SECRET)%3B%0A%20%20%20%20%7D%0A%7D%0A
+- title: Lebensdauern in Datentypen
+  content_markdown: >
+    Ähnlich wie bei Funktionen können Datentypen mit der Lebensdauer parametrisiert werden
+    Spezifizierer seiner Mitglieder.
+
+
+    Rust validiert, dass die enthaltende Datenstruktur der Referenzen niemals vorhanden ist
+    hält länger als die Besitzer, auf die seine Referenzen hinweisen.
+
+
+    Wir können keine Strukturen herumlaufen lassen, auf die Referenzen verweisen
+    Nichts!
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%3C'a%3E%20%7B%0A%20%20%20%20i%3A%26'a%20i32%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20x%20%3D%2042%3B%0A%20%20%20%20let%20foo%20%3D%20Foo%20%7B%0A%20%20%20%20%20%20%20%20i%3A%20%26x%0A%20%20%20%20%7D%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo.i)%3B%0A%7D%0A
+- title: Capitolul 5 - Abschluss
```

Translate "Capitolul" to German:
```suggestion
- title: Kapitel 5 - Abschluss
```